src/dvclive/lightning.py

```diff
@@ -113,6 +115,15 @@ def log_metrics(self, metrics: Dict[str, Any], step: Optional[int] = None):
                 self.experiment._latest_studio_step -= 1
             self.experiment.next_step()
 
+    def _log_checkpoint(self, checkpoint_callback):
+        self.experiment.log_artifact(checkpoint_callback.dirpath)
```

Comment:
I couldn't really find a way to log the single best model. Lightning by default uses filename patterns so we would be logging different files unless the user hardcodes a fixed pattern.

At this point, I am not sure if we would be better just documenting the context manager pattern and let the user decide what to log outside the callback.